crates/sdk/examples/quickstart-chat/main.rs

```diff
@@ -16,6 +16,7 @@ fn main() {
     connect_to_db();
     subscribe_to_tables();
     user_input_loop();
+    spacetimedb_sdk::global_connection::with_connection_mut(|x| x.disconnect())
```

Comment:
We should expose a global function that does this. `with_connection_mut` is supposed to be an internal detail.